lib/cart/models/cart.dart

```diff
@@ -0,0 +1,160 @@
+import 'package:flex_storefront/product_list/models/product.dart';
+import 'package:json_annotation/json_annotation.dart';
+
+part 'cart.g.dart';
+
+@JsonSerializable(createToJson: false)
+class Cart {
+  final String type;
+  final List<dynamic> appliedOrderPromotions;
+  final List<dynamic> appliedProductPromotions;
+  final List<dynamic> appliedVouchers;
+  final String code;
+  final int? deliveryItemsQuantity;
+  final List<CartItem> entries;
+  final String guid;
+  final bool net;
+  final int? pickupItemsQuantity;
+  // final CartDiscount productDiscounts;
+  final CartPrice subTotal;
+  final CartPrice totalDiscounts;
+  final int totalItems;
+  final CartPrice totalPrice;
+  final CartPrice totalPriceWithTax;
+  final CartPrice totalTax;
+  final User user;
+  final PaymentType? paymentType;
+  final List<dynamic> potentialOrderPromotions;
+  final List<dynamic> potentialProductPromotions;
+  final int totalUnitCount;
```

Comment:
Do you mind lightening this model and submodels a bit? Having just the fields that are actually used in the view makes the code base easier to work with.